### DISCUSSION TITLE
How to globally filter columns containing nested objects of data?

### DISCUSSION THREAD

**Discussion by akmur** [TanStack/table#3032] [Q&A]
Hello community,

I need to understand how I can go about filtering a column that contains an array of objects. 

I have implemented an example here:

https://codesandbox.io/s/react-table-basic-demo-with-header-groups-sort-filter-and-pagination-zoh4u?file=/src/components/columns.js

In this example, I have an object like:

```
{
    "id": 1,
    "actions": [
      {
        "type": "test",
        "tags": ["actiontag"]
      },
      {
        "type": "alex",
        "tags": ["yup"]
      }
    ],
    "first_name": "Marlene",
}
```

I am able to globally filter it by the type of actions through playing around with the accessor:

```
accessor: (row) => row.actions.map((a) => a.type).join(" "),
```

However I wonder how i would go about filtering with more complex objects (for example, a list of tags containing attributes). How would you approach this?

Thanks for your time

**Comment by 5no0p** [comment]
Did you figure out how to filter array of objects?
